variables.tf

```diff
@@ -811,3 +811,15 @@ variable "tcp_keepalive_probes" {
   type        = number
   default     = 2
 }
+
+variable "deny_global_write_access" {
+  description = "Deny all write permissions from the S3 bucket except producer_roles."
+  type        = bool
+  default     = false
+}
+
+variable "producer_roles" {
```

Comment:
how is this different from: `common_producer_roles`: https://github.com/ExpediaGroup/apiary-data-lake/blob/f4ce7639273f90d4c845c58d6fbfd4bf5aa61a55/variables.tf#L761C11-L761C42

Reply:
Common producer roles are used added across all buckets. `producer_roles` are individual for each bucket.

Reply:
Let's add update the: https://github.com/ExpediaGroup/apiary-data-lake/blame/master/VARIABLES.md